supabase/functions/sign-in-with-google/index.ts

```diff
@@ -0,0 +1,43 @@
+import { serve } from 'https://deno.land/std@0.131.0/http/server.ts'
+import { createClient, SupabaseClient } from 'https://esm.sh/@supabase/supabase-js@2'
+
+const corsHeaders = {
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Headers': 'authorization, x-client-info, apikey',
+}
+
+async function singInWithGoogle(supabaseClient: SupabaseClient) {
```

Comment:
I'm not familiar with Supabase: should we also enable `[auth.external.google]` in config.toml for this to work?

Reply:
@hernanr99 may know this